src/Plugin.php

```diff
@@ -84,4 +85,18 @@ public function console(CommandCollection $commands): CommandCollection
             ->add('queue requeue', RequeueCommand::class)
             ->add('queue purge_failed', PurgeFailedCommand::class);
     }
+
+    /**
+     * Add DI container to Worker command
+     *
+     * @param \Cake\Core\ContainerInterface $container The DI container
+     * @return void
+     */
+    public function services(ContainerInterface $container): void
+    {
+        $container->add(ContainerInterface::class, $container);
```

Comment:
Should we do this in CakePHP itself as well? This seems like an elegant solution for other framework/plugin abstractions to leverage the DI container.

Reply:
yea, this would be a nice default 😄